.storybook/main.ts

```diff
@@ -0,0 +1,51 @@
+import type { StorybookConfig } from '@storybook/react-webpack5'
+
+const config: StorybookConfig = {
+  stories: ['../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+    'storybook-addon-react-router-v6',
+    '@storybook/addon-viewport',
+  ],
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {},
+  },
+  core: {
+    disableTelemetry: true,
+    enableCrashReports: false,
+  },
+  babel: async options => ({
+    ...options,
+    presets: [
+      ...options.presets!,
+      ...[
+        '@babel/preset-flow',
```

Comment:
Do we need flow?

Reply:
we don't. not sure why I've encountered a build error with missing preset-flow yesterday